modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/310_reroute_processor.yml

```diff
@@ -139,3 +139,66 @@ teardown:
           query:
             match: {"_id": "example-log"}
   - match: { hits.hits.0._source.message: "this is an error log" }
+
+---
+"Test pipeline run after reroute":
```

Comment:
With the fix to the previous test, that test does cover what this is testing. But I still think it's worth having this simpler test to confirm the behavior.